Podfile.lock

```diff
@@ -487,7 +487,7 @@ PODS:
     - WordPressKit (~> 4.18-beta)
     - WordPressShared (~> 1.12-beta)
     - WordPressUI (~> 1.7-beta)
-  - WordPressKit (4.51.0-beta.1):
+  - WordPressKit (4.51.0):
```

Comment:
This is exactly why I opened https://github.com/wordpress-mobile/release-toolkit/issues/216 a while ago (wow… more than a year ago now 😓 ). I never found time to get to it, but maybe this is a good reminder that we should tackle this sooner than later?